service/service_test.go

```diff
@@ -203,6 +203,95 @@ func TestCreateArtifactType(t *testing.T) {
 	}
 }
 
+func TestUpdateArtifactType(t *testing.T) {
```

Comment:
Can you also write this unit test as integration test? The integration script is at

https://github.com/kubeflow/metadata/blob/master/test/e2e/make_requests.sh

The presubmit  job will start a server and make these requests.

Reply:
Yes. Of course. Thanks for the comment.